git.go

```diff
@@ -67,6 +67,15 @@ func (g *GitClient) Init(branch string) error {
 	if err := g.command("git", "config", "user.email", "concourse@local").Run(); err != nil {
 		return fmt.Errorf("failed to configure git email: %s", err)
 	}
+
+	if err := g.command("git", "config", "--global", "url.https://"+g.AccessToken+":x-oauth-basic@github.com/.insteadOf", "git@github.com:").Run(); err != nil{
```

Comment:
I think this line will write the access token to disk, which is something I'd like to avoid at all cost. Is there another way to make this work?

Reply:
I've tried to do that at runtime, but git doesn't have option to specify config at command execution time.